maxim/logger/langchain/utils.py

```
@@ -40,6 +40,18 @@ def parse_langchain_provider(serialized: Dict[str, Any]):
         return "bedrock"
     elif "aws" in provider:
         return "bedrock"
+    elif "openai" in provider:
+        return "openai"
+    elif "groq" in provider:
+        return "groq"
+    elif "ollama" in provider:
+        return "ollama"
+    elif "gemini" in provider or "vertexai" in provider:
+        return "google"    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)